azure-pipelines/util/build_appimage.sh

```diff
@@ -11,8 +11,13 @@ if [ -z "$APPDIR" ]; then
     exit 1
 fi
 
-# linuxdeploy location
-LINUXDEPLOY=${LINUXDEPLOY:-"linuxdeploy.AppImage"}
+# get linuxdeploy and linuxdeploy-plugin-gtk
+# Currently broken: wget -c "https://raw.githubusercontent.com/linuxdeploy/linuxdeploy-plugin-gtk/master/linuxdeploy-plugin-gtk.sh", see https://github.com/linuxdeploy/linuxdeploy-plugin-gtk/issues/7
+wget -c "https://raw.githubusercontent.com/linuxdeploy/linuxdeploy-plugin-gtk/c008df652946408f357f502bab67bfcf6f303b4e/linuxdeploy-plugin-gtk.sh"
+wget -c "https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage"
```

Comment:
This should check if the file has not been downloaded before trying to download it.